lazyflow/utility/pathHelpers.py

```diff
@@ -327,11 +329,11 @@ def mkdir_p(path):
             raise
 
 
-def lsHdf5(hdf5FileObject, minShape=2, maxShape=5):
-    """Generates dataset list of given h5py file object
+def lsH5N5(h5N5FileObject, minShape=2):
```

Comment:
Ah, sorry I wanted to ask you that and forgot it. I will undo it. I removed it here because the other function (which i removed) _get_hdf5_dataset_names() in opInputDataReader.py also did not have it...